fitbenchmarking/fitbenchmarking
# Issue #446: Add autogenerated module index

#### Description of Work

Fixes #446 

Makes module_index build automatically with sphinx docs.

#### Testing Instructions

1. Check the docs worked ([here](https://fitbenchmarking.readthedocs.io/en/446-module_index/))

Function: Does the change do what it's supposed to?
Tests: Does it pass? Is there adequate coverage for new code?
Style: Is the coding style consistent? Is anything overly confusing?
Documentation: Is there a suitable change to documentation for this change?
